eval.go

```diff
@@ -359,9 +359,8 @@ func exec(fr *Frame, stmt syntax.Stmt) error {
 		}
 		if cond.Truth() {
 			return execStmts(fr, stmt.True)
-		} else {
-			return execStmts(fr, stmt.False)
```

Comment:
Please leave as is.  The common indentation makes the symmetry of the two branches more obvious.  I don't mind indenting the error case and not the normal case, but that rule needn't apply when the two cases are equally normal.